lib/order1.rb

```diff
@@ -0,0 +1,41 @@
+require './lib/menu.rb'
+
+class Order1
```

Comment:
Is this another implementation of the same Order class from above? 🙂 

I actually prefer this version as I think the implementation of the `add_item` method is much cleaner. I also like that it delegates some responsibility down to the `Menu` class.